### DISCUSSION TITLE
__nuxt, __layout, vue3 fragments

### DISCUSSION THREAD

**Discussion by stratboy** [nuxt/nuxt#8586] [Questions (Nuxt 2)]
Hi, 2 basic questions:

- is there any way to get rid of the 2 container divs #__nuxt and #__layout?

- I guess Nuxt is currently based on Vue 2, which always needs (often useless) containers for templates. Vue3 has fragments, which I love. Any chance you will soon implement Vue3 and its Fragments feature?

Thank you, regards

**Comment by B-starboy** [comment]
Yes we are going to integrate fragments Vue 3

**Reply by stratboy** [reply]
Sorry I switched to Next. 24 days are quite too much to show out good customer support and mature project development ;)

Maybe I'll come back later because I love Vue... But you must prove that the NUxt project is as up to date as Next. Nuxt is very good, but Next is currently more mature and has stronger and faster support and docs, which to me, is essential.

Thank you anyway,

Regards
